examples/08_manage_a_cart/06_update_cart_discount_codes/variables.json

```diff
@@ -0,0 +1,6 @@
+{
+	"cartId": "Z2lkOi8vc2hvcGlmeS9DYXJ0L2QzNTNhODQxYjQ1Y2VmYjY3ZTQ1NjdiZGUzMzU4MjFh",
```

Comment:
With base64 encoded ids being deprecated, this could be confusing to users. Is it possible for the example to use non-encoded ids?